torch/_dynamo/variables/tensor.py

```diff
@@ -498,7 +498,43 @@ def make_const_size_variable(x, **options):
                 example_value=None,
                 **options,
             )
-        elif name in ("tolist", "backward", "data_ptr"):
+        elif name == "tolist":
+            if self.size and free_symbols(self.size):
+                unimplemented(f"Tensor.{name}")
+            else:
+                from .builder import SourcelessBuilder
+
+                def tolist(tensor, sub_proxy):
+                    def wrap(i, sub_proxy):
+                        return SymNodeVariable.create(
+                            tx,
+                            sub_proxy.item(),
+                            sym_num=tx.output.shape_env.create_unbacked_symint(),
+                        )
+
+                    if tensor.dtype not in [
+                        torch.int8,
+                        torch.int16,
+                        torch.int32,
+                        torch.int64,
+                    ]:
+                        unimplemented(
+                            "Input tensor for tolist must be an integer tensor"
```

Comment:
In the long term, tolist as float should work, since we'd model each entry as a scalar float tensor and translate Python float operations into their tensor-y equivalents.

Reply:
could we just do the same but w/ `create_unbacked_symfloat`?

Reply:
Inconsistent with our design policy, see https://docs.google.com/document/d/1INSCdYu1PxXcr43HrD82OudeEuS-qxQe1yZmLg2wy6A/edit

Reply:
Oh thats right